app/models/video.rb

```diff
@@ -1,9 +1,19 @@
 class Video < ActiveRecord::Base
   belongs_to :category
+  has_many :reviews, -> { order "created_at DESC" }
+  has_many :queue_items
 
   validates_presence_of :title, :description
 
   def self.search_by_title(search_term)
     search_term.blank? ? [] : where("title ILIKE ?", "%#{search_term}%").order(created_at: :desc)
```

Comment:
Agree, but not sure I understand how your comment applies to the code. The ``` if ``` statement was to cover the case when the user searches with an empty string, which would otherwise return all of the videos with ``` .where ```.